### PR TITLE
Feature milestone 3 -> Style

### PR DESCRIPTION
**Pull request by santiagorodriguezbermudez**
# Milestone 3 -> Style

### On this milestone I implemented:
- [x] Refactor code per TSE recommendation
- [x] Styled of the App component
- [x] Styled for display component
- [x] Styled for the Button Panel component
- [x] Added props for Button styles

### CODE REVIEW

File: src/components/ButtonPanel.jsx [hunk 1]
```diff
@@ -2,39 +2,30 @@ import React from 'react';
...
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+
+  const listedButton = (btnGroup => btnGroup.map(button => {
+    let buttonComponent;
+    if (['÷', 'X', '-', '+'].includes(button)) {
```
Comment (bellom) [inline src/components/ButtonPanel.jsx]: we are currently having this 👇 ;
image

`=` should be in orange color too.

we need to add `=` this array. includes right? 

And also, let's make use of lowercase `x` the current sign is looking way bigger than the rest of the signs.

File: src/components/ButtonPanel.jsx [hunk 2]
```diff
@@ -2,39 +2,30 @@ import React from 'react';
...
-        <Button name="+" />
-      </div>
-      <div id="group5">
-        <Button name="0" />
-        <Button name="." />
-        <Button name="=" />
-      </div>
+  const symbolArray = [
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', 'X'],
```
Comment (bellom) [inline src/components/ButtonPanel.jsx]: Let's make use of lowercase `x` the current sign is looking way bigger than the rest of the signs.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @santiagorodriguezbermudez  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/components/ButtonPanel.jsx (hunk 1 above)
Comment (santiagorodriguezbermudez) [reply src/components/ButtonPanel.jsx]: Yes, you're right. I'm on it.

**Comment by santiagorodriguezbermudez** [comment]
## Code Review # 1

### Per TSE advice, I worked on:
- [x] Change the color to the '=' button. 
- [x] Change the case of the 'x' button.

**Review by bellom — APPROVED** [review]
## Hi @santiagorodriguezbermudez  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**